src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -647,10 +647,6 @@ private function getUserProvider(ContainerBuilder $container, string $id, array
             return $providerIds[$normalizedName];
         }
 
-        if ('remember_me' === $factoryKey && $contextListenerId) {
```

Comment:
I guess we then also can remove the `$contextListenerId` from the method parameters

Reply:
You're right. It's updated !